src/vcpkg/sourceparagraph.cpp

```diff
@@ -857,6 +857,7 @@ namespace vcpkg
         constexpr static StringLiteral SUPPORTS = "supports";
         constexpr static StringLiteral OVERRIDES = "overrides";
         constexpr static StringLiteral BUILTIN_BASELINE = "builtin-baseline";
+        constexpr static StringLiteral VCPKG_CONFIGURATION = "x-vcpkg-configuration";
```

Comment:
```suggestion
        constexpr static StringLiteral VCPKG_CONFIGURATION = "configuration";
```

IMO, since this isn't starting as a completely unknown object form etc. I don't think this needs to begin life as an x- thing?

Reply:
I thought that as long as it is experimental (behind non-default feature flag) the `x-` prefix was needed.